src/request/request.service.ts

```diff
@@ -98,16 +141,43 @@ export class RequestService {
 
     // 1 because to get team name from description
     const teamName = input.description.split(' ')[1];
+    const player = input.description.split(' ')[4];
```

Comment:
put an example of string here, or a template, like `<team> <smth> <smth2> <player>`

Reply:
what governs the description? is player supposed to input it manually?

Reply:
this endpoint is for manager or administrator. And backend takes the description, but assuming that the manager gets it from another endpoint